package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "platform-custom-scalars",
+  "version": "1.0.0",
+  "private": false,
+  "description": "A library of customized GraphQL scalars for Ontotext Platform",
+  "homepage": "https://github.com/Ontotext-AD/platform-custom-scalars#readme",
+  "author": {
+    "name": "'Sirma AI' JSC, trading as Ontotext",
+    "url": "https://www.ontotext.com/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Ontotext-AD/platform-custom-scalars.git"
+  },
+  "keywords": [
+    "platform",
+    "scalars",
+    "graphql"
+  ],
+  "dependencies": {
+    "graphql": "^15.0.0",
+    "bignumber.js": "^9.0.0",
+    "@jest/globals": "^26.0.1",
```

Comment:
Maybe all but `globals` jest stuff can be a dev dependency

Reply:
Done.